src/services/createNovel.js

```diff
@@ -0,0 +1,49 @@
+import models from '../database/models';
+
+const { Genre, Novel } = models;
+
+/**
+ *
+ *
+ * @param {object} novel
+ * @param {object} author
+ * @returns {object} json
+ */
+const createNewNovel = async (novel, author) => {
+  const {
+    genre,
+    title,
+    description,
+    body
+  } = novel;
+  const getGenre = await Genre.findOne({ where: { name: genre } });
+  const slug = `${title.toLowerCase().split(' ').join('-')}-${author.id}`;
```

Comment:
why are we appending the author id to the slug? the slug should be unique

Reply:
It is unique, the reason for the author id is to allow for different users to create novels with the same name but to limit the user from uploading a novel twice or with the same name

Reply:
ok